rct229/rulesets/ashrae9012019/section5/section5rule34.py

```diff
@@ -52,20 +52,20 @@ def get_calc_vals(self, context, data=None):
             matched_baseline_zones = match_lists_by_id(proposed_zones, baseline_zones)
             proposed_baseline_zone_pairs = zip(proposed_zones, matched_baseline_zones)
             for p_zone, b_zone in proposed_baseline_zone_pairs:
-                # need a method like match object
-                p_zone_infiltration = p_zone["infiltration"]
-                # b_zone could be NONE - add a check.
-                b_zone_infiltration = b_zone["infiltration"]
+                if b_zone is not None:
```

Comment:
what is happening if b_zone is `None`?

Reply:
@weilixu @yunjoonjung-PNNL I think the case where a match cannot be made between baseline and proposed zones needs to be handled explicitly as an undetermined case.

Reply:
Addressed in PR #1593.